packages/simple-markdown/simple-markdown.js

```diff
@@ -0,0 +1,2111 @@
+(function (global, factory) {
```

Comment:
This looks like the generated file.  This should be in `dist/index.js` (you can check the `package.json` for `packages/perseus` and mimic how it defines three different files. Our rollup build will build the dist files that are needed